astropy/table/tests/test_operations.py

```diff
@@ -828,8 +830,8 @@ def _setup(self, t_cls=Table):
     def test_default_same_columns(self, operation_table_type):
         self._setup(operation_table_type)
         out = table.setdiff(self.t1, self.t2)
-        assert type(out['a']) is type(self.t1['a'])  # noqa
-        assert type(out['b']) is type(self.t1['b'])  # noqa
+        assert isinstance(out['a'], type(self.t1['a']))
```

Comment:
I'd restore a noqa to these types of tests. The ``isinstance`` replacement is less strict a comparison than the ``is``.

Reply:
There are a lot more of these issues scattered throughout `astropy` (we don't catch them because we don't acutally run our entire `flake8` config).

So instead of adding a bunch more `# noqa` flags I added a helper function which implements the "type is type" assertion (strangely `flake8` is fine with that one?). I did this in fb658d8, do you think this is better?

Reply:
I think so! Might need a changelog entry for it

Reply:
Just added one!